alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,77 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+Custom Resources managed by Prometheus Operator were rejected and not propagated
+to operated Prometheus and Alertmanager.
+The alert `PrometheusOperatorRejectedResources` is triggered by misconfiguration
+of these custom resources:
+- Alertmanager
+- PodMonitor
+- Probe
+- PrometheusRule
+- ScrapeConfig
+- ServiceMonitor
+
+## Impact
+
+The custom resource triggering the alert will be ignored by Prometheus Operator,
+and by consequence will not be integrated into the configuration of the Prometheus
+and the Alertmanager managed by Prometheus Operator.
+
+## Diagnosis
+
+Find out the type of custom resource and its namespace in the `description` field
+of the alert `PrometheusOperatorRejectedResources`.
```

Comment:
Yup, I will add `If the namespace is named `openshift-.*`, for example, `openshift-monitoring`,
this may be a bug in Openshift, please file a support case.`